### ISSUE TITLE
Wizard previous and next buttons focus style

### ISSUE THREAD

**Issue by afercia** [Yoast/yoast-components#833]
The new previous / next buttons introduced in #831 don't have a focus style in Firefox. They rely on the browsers native outline, and in Firefox the outline doesn't work when unsetting the borders.

To reproduce:
- use Firefox (mac or Windows)
- tab to the previous and next buttons:

![screenshot 2019-02-27 at 11 07 21](https://user-images.githubusercontent.com/1682452/53482668-0ffd6380-3a80-11e9-8f41-d72f9d09f68e.png)

- there's no focus style, this prevents users from understanding where focus is
